### ISSUE TITLE
imp3

### ISSUE THREAD

**Issue by THESAURABH07** [THESAURABH07/data-main#3]
5.Write a program to Visualize the dataset to gain
insights using Matplotlib
#or Seaborn by plotting scatter plots, bar charts.
import numpy as np
import pandas as pd
from sklearn.preprocessing import StandardScaler
from sklearn.preprocessing import MinMaxScaler
from sklearn.preprocessing import RobustScaler
import matplotlib.pyplot as plt
import seaborn as sns
data={'data1':[10,20,30,40,50],'data2':[1200,1500,3
500,4000,8000]}
print("data is ",data)
df=pd.DataFrame(data)
print(df)
plt.figure(1)
plt.bar(df['data1'],df['data2'])
plt.title('data before scaler with plt')
plt.xlabel('data1')
plt.ylabel('data2')
plt.figure(2)
plt.scatter(data['data1'], data['data2'], color='blue',
alpha=0.7, edgecolors='black')
# Labels and title
plt.xlabel('Data 1')
plt.ylabel('Data 2')
plt.title('Scatter Plot before scaler using
Matplotlib')
rs=RobustScaler()
rdata=rs.fit_transform(df)
dfr=pd.DataFrame(rdata,columns=df.columns)
print(dfr)
plt.figure(3)
sns.set_theme(style="darkgrid")
sns.barplot(x='data1',y='data2',data=dfr)
plt.title('data after scaler with sns')
plt.figure(6)
sns.scatterplot(data=dfr, x='data1', y='data2',
color='red')
plt.title('Scatter Plot after scaler using Seaborn')
plt.show()
